extension/test/data_test.ts

```diff
@@ -69,4 +69,209 @@ describe('data.ts', function () {
       ).to.include.something.like({ entry: /^ＤＶＤ-ＲＯＭ .*/ });
     });
   });
+
+  describe('makeText', function () {
+    describe('with a non-kanji dict entry', function () {
+      const unimportantProperties = {
```

Comment:
Two things:
1.  Usually, we'd define this outside the test as a module local constant so that it's not taking up space while reading the test. The idea is that if the defaults are truly unimportant then they don't need to be seen while reading.

2.  Related, if you move it outside you'd probably want one that worked for all your tests. In that sense, instead of specifying the exact unimportant properties, you would just have a `DEFAULT_DICT_ENTRY` or similar and use that anywhere you needed to customize.